src/pkg.js

```diff
@@ -57,20 +55,23 @@ const updateVersion = ({ path, packageName, logger }) => (
             peerDependencies = {}
         }, version) => {
             if (dependencies.hasOwnProperty(packageName)) {
+                assert(!satisfies(version, dependencies[packageName]), `${packageName} latest version ${version} matches existing dependency ${dependencies[packageName]}`);
```

Comment:
Is this really something you never expect to happen?  Seems like a weird use of assert, but maybe I'm missing the code that would have prevented this